### PR TITLE
feature/menu redesign

### PR DESCRIPTION
**Pull request by reflash**
Navbar state code became redundant, so I've removed it.
Both desktop and mobile version are now displaying the links as blocks.

## Desktop
image

## Mobile
image

Also now task page will have an arrow button to return back:
image

### CODE REVIEW

File: src/Client/Markup.fs [hunk 1]
```diff
@@ -15,17 +15,15 @@ type Clickability =
 
 let goToUrl (e: React.MouseEvent) =
     e.preventDefault()
-    let href = !!e.target?href
+    let href = !!e.currentTarget?href
```
Comment (reflash) [inline src/Client/Markup.fs:18]: target will point to an html element that was clicked, not the <a> element. So when I have nested elements in <a> and click on a <span>: target will contain <span>?href = undefined; currentTarget will contain <a>?href = correctUrl

**Comment by reflash** [comment]
deployed to experimental

File: src/Client/Markup.fs (hunk 1 above)
Comment (psfinaki) [reply src/Client/Markup.fs:18]: Alright :)

File: src/Client/pages/Home.fs [hunk 2]
```diff
@@ -7,6 +7,13 @@ open Fulma
 open Client
 open Client.AppPages
 
+let linkBlock text colorClass = 
+    let firstSymbol: char = Seq.head text
```
Comment (psfinaki) [inline src/Client/pages/Home.fs:11]: Maybe just `let firstSymbol = text.[0]`? [Ultimate recommendation](https://stackoverflow.com/a/3878828/3232646) :D

File: src/Client/Markup.fs [hunk 3]
```diff
@@ -15,17 +15,15 @@ type Clickability =
...
-let viewLinkCentered className page description =
+let viewLinkCentered className page children =
     a [ 
         ClassName className
         Href (toHash page)
         OnClick goToUrl 
-    ] [ 
-        str description
-    ]
+    ] children
```
Comment (psfinaki) [inline src/Client/Markup.fs:26]: So what are examples of children here?

File: src/Client/widgets/Navbar.fs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+module Client.Widgets.Navbar
+
+open Fable.Helpers.React
+open Fable.Helpers.React.Props
+open Fulma
+
+
```
[the remark this answers is not in the document]
Comment (reflash) [reply src/Client/widgets/Navbar.fs]: Removed

File: src/Client/pages/Home.fs (hunk 2 above)
Comment (reflash) [reply src/Client/pages/Home.fs:11]: using head seems more idiomatic to me (for f#) 😄

File: src/Client/Markup.fs (hunk 3 above)
Comment (reflash) [reply src/Client/Markup.fs:26]: a list of elements (other components, html elems, etc)

File: src/Client/pages/Home.fs (hunk 2 above)
Comment (psfinaki) [reply src/Client/pages/Home.fs:11]: Alright

**Pull request merged**